tests/user_restricted/configuration/config.py

```diff
@@ -1,56 +0,0 @@
-
```

Comment:
if this file is deleted, a) where is config for user_restricted pulled from? b) if it pulled from somewhere else (that's fine), but can the directory be deleted if nothing in it?

Reply:
I think it is: it isn't appearing in my branch; it just shows up on the diff.